packages/koa-shopify-auth/src/auth/create-enable-cookies.ts

```diff
@@ -40,7 +40,7 @@ export default function createEnableCookies({
 
   <script>
     window.apiKey = "${apiKey}";
-    window.shopOrigin = "https://${shop}";
+    window.shopOrigin = "https://${encodeURIComponent(shop)}";
```

Comment:
Should the `https://` be part of the `encodeURIComponent` call as well?

Reply:
i'm pretty sure "shop" always comes back as foo.myshopify.com

a couple years ago, @ragalie did this

https://github.com/Shopify/quilt/commit/3284be1e6cfe9165a2b23339d7f1bf8367ee9df7#diff-a15b7c0928331371cc40785dd637c644R16

which is a different way of fixing this, in a different place

Reply:
🤔 I think it's okay since we only want to encode the value coming from the user

Reply:
That makes sense 👍